gnx.js

```diff
@@ -335,6 +335,11 @@ const buildInputType = function (gqltype) {
   return { inputTypeBody: inputTypeForAdd, inputTypeBodyForUpdate: inputTypeForUpdate }
 }
 
+const getInputType = function (type) {
+  return typesDict.types[type.name].inputType
+}
+exports.getInputType = getInputType
```

Comment:
I think is `module.exports` instead of `exports`

Reply:
fixed